apps/tmbitwarden.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2018, https://www.hanssonit.se/
+
+echo "Installing Bitwarden password manager..."
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Check if root
+root_check
+
+msg_box "Bitwarden is a password manager that is seperate from Nextcloud, though we provide this service because it's self hosted and secure.
+
+If you just want to run Bitwarden locally (not connecting your smartphone) then you can use 'localhost' as domain.
+If you on the other hand want to run this on a domain, then please create a DNS record and point it to this server.
+Well add support for SSL in a later version of this script, but for now you have to do that stuff manually.
+
+The script is based on this documentation: https://help.bitwarden.com/article/install-on-premise/
+It's a good idea to read that before you start this script."
+
+# Test RAM size (2GB min) + CPUs (min 2)
+ram_check 2 Bitwarden
+cpu_check 2 Bitwarden
```

Comment:
You should be ok with just 1 CPU.

Reply:
Yeah for Bitwarden itself, yes. This VM runs more applications than that. 

Thanks for the advice though, keep em coming :)